src/components/Header/LongHeader.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import { BackgroundBox } from '@/components/BackgroundBox';
+import { Button } from '@/components/Button';
+import { Input } from '@/components/Input';
+import { Image } from '@/components/Image';
+import MenuIcon from '@mui/icons-material/Menu';
+import SearchIcon from '@mui/icons-material/Search';
+import NotificationsActiveIcon from '@mui/icons-material/NotificationsActive';
+import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
+import { getImageSrc } from '@/utils/helpers';
+
+const LongHeader = ({ isLoggedIn }) => {
+  return (
+    <Wrapper>
+      <BackgroundBox borderRadius="0 0 1.6rem 1.6rem" height="17rem">
+        <TopContainer>
+          <StyledMenu />
+          <h1>Comepet</h1>
+          {isLoggedIn ? (
+            <IconContainer>
+              <StyledInformation />
+              <StyledNotification />
+            </IconContainer>
+          ) : (
+            <Button
+              bgColor="#fff"
+              type="button"
+              width="2.8rem"
+              height="2.8rem"
+              borderRadius="50%"
+              margin="0 1rem 0 0">
+              <Image alt="로그인" width="2.8rem" height="2.8rem" type="profile" />
+            </Button>
+          )}
+        </TopContainer>
+        <MiddleContainer>
+          <BackgroundBox width="15rem" boxShadow="0px 4px 16px rgba(0, 0, 0, 0.08)">
+            <Button
+              bgColor="#fff"
+              type="button"
+              color="#2A2E56"
+              fontWeight="bold"
+              borderRadius="1.6rem">
+              <Image
+                src={getImageSrc('/images/finding.png')}
+                width="1.6rem"
+                height="1.6rem"
+                margin="0 5rem 0 0"
+              />
+              실종 및 보호
+            </Button>
+          </BackgroundBox>
+          <BackgroundBox width="15rem" boxShadow="0px 4px 16px rgba(0, 0, 0, 0.08)">
+            <Button
+              bgColor="#fff"
+              type="button"
+              color="#2A2E56"
+              fontWeight="bold"
+              borderRadius="1.6rem">
+              <Image src={getImageSrc('/images/home.png')} width="1.6rem" height="1.6rem" />
+              보호소 동물
+            </Button>
+          </BackgroundBox>
+        </MiddleContainer>
+        <BottomContainer>
+          <Input placeholder="세부 검색을 위해 클릭해주세요" borderRadius="1.6rem" />
+          <StyledSearch />
```

Comment:
Input의 Icon 수직 정렬에 대한 해결법입니다.

```suggestion
          <InputWrapper>
            <Input placeholder="세부 검색을 위해 클릭해주세요" borderRadius="1.6rem" />
            <StyledSearch />
          </InputWrapper>
```